src/test/java/org/folio/rest/impl/OkapiMockServer.java

```diff
@@ -438,6 +446,22 @@ private void handleRecordStorageResultGetResponse(RoutingContext ctx) {
     }
   }
 
+  private String getSrsRecordsPartially(int offset, int limit) {
+    String sourceRecordsString = requireNonNull(getJsonObjectFromFile(SOURCE_STORAGE_RESULT_URI + INSTANCES_10_TOTAL_RECORDS_10));
+    String srsRecordsResponseTemplate = requireNonNull(getJsonObjectFromFile(SOURCE_STORAGE_RESULT_URI + SRS_RESPONSE_TEMPLATE_JSON));
+    JsonObject sourceRecordsJson = new JsonObject(sourceRecordsString);
+    JsonArray defaultRecords = sourceRecordsJson.getJsonArray("sourceRecords");
+    JsonArray requiredRecords = new JsonArray();
+    int interval = offset == 8 ? 2 : limit;
```

Comment:
What is 8 here means? could you extract it to a constant and name it properly?

Reply:
Values moved to constants.